lib/hmmlearn/hmm.py

```diff
@@ -232,11 +240,8 @@ def _initialize_sufficient_statistics(self):
                                            self.n_features))
         return stats
 
-    def _accumulate_sufficient_statistics(self, stats, obs, framelogprob,
-                                          posteriors, fwdlattice, bwdlattice):
-        super()._accumulate_sufficient_statistics(
-            stats, obs, framelogprob, posteriors, fwdlattice, bwdlattice)
```

Comment:
what happened here?

Reply:
The update step for a single iteration is now done in two separate steps:
1. Update the transition / start probabilities using a separate method for scaling / logarithms)
2. Update the emission probabilities using a common method for scaling / logarithms
This replaces the single method, that used inheritance / `super()` to update all probabilities in one go.

Reply:
Ah, I see.  See comment above for perhaps re-merging them together and adding an internal `if self.implementation == "scaling": ... elif == "log": ...`